src/get-started/updating-your-code/index.md

```diff
@@ -190,10 +220,15 @@ The main change is changing variables from [snake_case](https://en.wikipedia.org
           You can now entirely replace the footer component by rendering your own,
           which can include custom <code>meta</code> links
         </p>
-        <pre><code>{% raw %}
+
+```javascript
+{% raw %}
 {% block header %}
-    {{ govukFooter({ meta: [] }) }}
-{% endblock %}{% endraw %}</code></pre>
+  {{ govukFooter({ meta: [] }) }}
```

Comment:
Same issue here!

Reply:
Also fixed in commit https://github.com/alphagov/govuk-design-system/pull/2578/commits/056cd10adb65be12f324314399704a6d2cbbce74